dotcom-rendering/cdk/lib/renderingStack.ts

```diff
@@ -179,14 +179,16 @@ export class RenderingCDKStack extends CDKStack {
 				},
 			);
 
-			const criticalAlertsTopic = Topic.fromTopicArn(
-				this,
-				'CriticalAlertsTopic',
-				`arn:aws:sns:${region}:${this.account}:Frontend-${stage}-CriticalAlerts`,
-			);
-			const criticalAlertsSnsAction = new SnsAction(criticalAlertsTopic);
+			const scalingAlertsTopic = new Topic(this, 'ScalingAlertsTopic');
+			new Subscription(this, 'ScalingAlertsSubscriptionEmail', {
+				endpoint: 'dotcom.platform@theguardian.com',
```

Comment:
We could use email labels, here:

```suggestion
				endpoint: 'dotcom.platform+aws.scaling@theguardian.com',
```